src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
Ah, I've missed that. No, those will need to be defined as they used to be for Unix, so the SOURCE_FORMATTING define introduction sounds like a good thing to do.